.github/workflows/ci.yml

```diff
@@ -42,9 +42,10 @@ jobs:
           maven-java-${{ matrix.java }}-
           maven-
     - name: Set up Java
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v2
       with:
         java-version: ${{ matrix.java }}
+        distribution: 'temurin'
```

Comment:
"temurin" doesn't work here because it doesn't provide the right Java versions. Use "zulu" instead.